code/drasil-sysinfo/lib/SysInfo/Drasil/SystemInformation.hs

```diff
@@ -74,16 +68,6 @@ type Scope = [Sentence]
 -- | Project Example motivation.
 type Motivation = [Sentence]
 
--- | Helper for extracting a bibliography from the system information.
-citeDB :: SystemInformation -> BibRef
-citeDB si = citationsFromBibMap (_citationDB (refdb si))
-
--- | Create sorted citations from a bibliography.
-citationsFromBibMap :: BibMap -> [Citation]
-citationsFromBibMap bm = sortBy compareAuthYearTitle citations
-  where citations :: [Citation]
-        citations = map fst (Map.elems bm)
-
 -- | Orders two authors. If given two of the exact same authors, year, and title, returns an error.
 compareAuthYearTitle :: (HasFields c) => c -> c -> Ordering
```

Comment:
I really really really want to move this function + `citeDB` out of this package entirely, which is why this PR is a draft.

Reply:
This was completed in 48a0e8028b0df7fda357dcbff0e9834462566a1f.